src/main/java/step5/domain/Cars.java

```diff
@@ -0,0 +1,54 @@
+package step5.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> carList;
+
+    public Cars(List<String> carNames) {
+        this.carList = create(carNames);
+    }
+
+    // 테스트 코드를 위한 생성자 (더미 매개변수를 받아서 해도 괜찮은 건지 모르겠습니다  ㅠ)
+    public Cars(List<Car> cars, int dummy) {
+        this.carList = cars;
+    }
+
+    public List<Car> getCarList() {
+        return carList;
+    }
+
+    private List<Car> create(List<String> carNames) {
+        List<Car> cars = new ArrayList<>();
+        for (int i = 0; i < carNames.size(); ++i) {
+            cars.add(new Car(carNames.get(i)));
+        }
+
+        return cars;
+    }
+
+    public void move() {
+        for (int i = 0; i < carList.size(); ++i) {
+            Car car = carList.get(i);
+            car.move(new RandomMovingStrategy());
+        }
```

Comment:
향상된 포문을 사용하시면 더 깔끔해질 것 같네요 👍 
```suggestion
        for (Car car : carList) {
            car.move(new RandomMovingStrategy());
        }
```